README.md

```diff
@@ -48,90 +54,107 @@ Make sure to run `npm ci` to update node packages.
 
 #### Next, Create a file named .env, copy the contents of the .env.template file into it, and add your values to the new file.
-- Paste the access token into GITHUB_TOKEN
-- TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Typically, this would either be "Etc/UTC" or the time zone of your organization's main office, such as "America/Los_Angeles". 
 
-If for some reason you want the time to be a constant offset from UTC, you can say : "Etc/GMT+6", to mean UTC-0600. **Note that positive values in the TIMEZONE string will result in negative UTC offsets** (that is, West of UTC), while **negative values will result in positive UTC offsets.** This is a result of IANA, not by choice. 
-For example: 
-``` .env
+-   Paste the access token into GITHUB_TOKEN
+-   TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Typically, this would either be "Etc/UTC" or the time zone of your organization's main office, such as "America/Los_Angeles".
+
+If for some reason you want the time to be a constant offset from UTC, you can say : "Etc/GMT+6", to mean UTC-0600. **Note that positive values in the TIMEZONE string will result in negative UTC offsets** (that is, West of UTC), while **negative values will result in positive UTC offsets.** This is a result of IANA, not by choice.
+For example:
+
+```.env
 TIMEZONE=“Etc/GMT+6”
 ```
 
 would output:
 
-
-``` 
-
-Users that contributed between Tue Mar 31 2020 00:00:00 GMT-0600 and Tue Apr 07 2020 23:59:59 GMT-0600 
 ```
 
+Users that contributed between Tue Mar 31 2020 00:00:00 GMT-0600 and Tue Apr 07 2020 23:59:59 GMT-0600
+```
 
 For further reading visit [moment-timezone](https://momentjs.com/timezone/docs) and [List of tz](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
-- The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed. For example, in the example CSV above, CSV_COLUMN_NUMBER_FOR_GITHUB_ID = 0 and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID = 1. If you choose not to use an alternate id, you can put the same column number in both CSV_COLUMN_NUMBER_FOR_GITHUB_ID and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID.
+
+-   The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed. For example, in the example CSV above, CSV_COLUMN_NUMBER_FOR_GITHUB_ID = 0 and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID = 1. If you choose not to use an alternate id, you can put the same column number in both CSV_COLUMN_NUMBER_FOR_GITHUB_ID and CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID.
+
+To filter out events for which the author is the OWNER of the repository, simply set the IGNORE_OWN_EVENTS environment variable to true or True.
```

Comment:
Thank you for updating the README to reflect the change you made in the code :-)
This sentence should be a bullet point like the others (for example, `- Paste the access token into GITHUB_TOKEN`) to make it easy to find.

Unfortunately, it looks like your file changes almost every line of the README. Maybe Starfish doesn't have linter rules set up for Markdown files, but your text editor does? I actually spent some time looking through the READMEs to see if there was any other content you changed, or if the two files look different, but I think all of these other changes are just how the code in the file itself looks. I don't want to  make everybody else working on Starfish have to update their code so please disable your linter or whatever you need to do so that the only change in the README file is the adding of your instructions.

Reply:
For what it's worth, I see a ton of linting issues when I load README into vscode. In a separate ticket, we should probably either do a one-pass edit of README to comply with the linting rules, OR change the linting rules to match the current README so there are no warnings OR add the README to the linter's ignore file.

Reply:
I created a new ticket for that, but then I thought-  do you think that means that we should actually keep all of these README changes because they're fixing those linting issues you're seeing? I'm not seeing them on my computer (though I'm currently in the Andre tests branch.)

Reply:
I would prefer for this MR to contain a minimum of README changes. Since you already have a README ticket planned, we should probably figure out the linting as part of that.